test/lib/browsers/playwright.ts

```diff
@@ -451,6 +451,10 @@ export class Playwright<TCurrent = undefined> {
     )
   }
 
+  hasElementsByCss(selector: string) {
+    return this.startChain(() => page.$$eval(selector, (els) => els.length > 0))
+  }
+
```

Comment:
This is unused. Let's not introduce test helpers until where's a wide-spread pattern